core/src/main/java/org/elasticsearch/index/query/FilteredQueryBuilder.java

```diff
@@ -83,9 +158,7 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
             builder.field("filter");
             filterBuilder.toXContent(builder, params);
         }
-        if (boost != -1) {
-            builder.field("boost", boost);
-        }
+        builder.field("boost", boost);
```

Comment:
We changed the boost default to 1.0f in the other builders, should we do this here to for consistency? Can't add this comment on the top where the default is set.